sfs/mono/source.py

```diff
@@ -79,6 +81,57 @@ def point_velocity(omega, x0, n0, grid, c=None):
     return util.XyzComponents([v * o / r for o in offset])
 
 
+def point_dipole(omega, x0, n0, grid, c=None):
+    """Point source with dipole characteristics.
+
+    Parameters
+    ----------
+    omega : float
+        Frequency of source.
+    x0 : (3,) tripple of array_like
+        Position of source.
+    n0 : (3,) tripple of array_like
+        Normal vector (direction) of dipole.
+    grid : triple of numpy.ndarray
+        The grid that is used for the sound field calculations.
+        See :func:`sfs.util.xyz_grid`.
+    c : float, optional
+        Speed of sound.
+
+    Returns
+    -------
+    numpy.ndarray
+        Sound pressure at positions given by `grid`.
+
+    Notes
+    -----
+    ::
+
+         d                1   / iw       1    \   (x-x0) n0
+        ---- G(x-x0,w) = --- | ----- + ------- | ----------- e^(-i w/c |x-x0|)
+        d ns             4pi  \  c     |x-x0| /   |x-x0|^2
+
+    Examples
```

Comment:
I would say so, it is also called "Examples" here:
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections